addons/http_routing/models/ir_http.py

```diff
@@ -650,7 +650,10 @@ def _handle_error(cls, exception):
         return response
 
     @api.model
-    @tools.ormcache('path', 'query_args')
+    @tools.ormcache_longterm(
+        'self.env["website"]._get_cache_longterm_key()',
```

Comment:
Website in http_routing? (didn't read the PR/code yet, just scrolled quickly)

Reply:
I put it in the wrong place. well seen